src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -228,81 +228,19 @@ function WebGLShadowMap( _renderer, _objects, _capabilities ) {
 
 	}
 
-	function getDepthMaterialVariant( useMorphing ) {
```

Comment:
@mrdoob It seems we don't need the `*MaterialVariant()` functions anymore 🎉 .

Reply:
And.... this now works with morphs!
```
scene.overrideMaterial = new THREE.MeshDepthMaterial( { depthPacking: THREE.RGBADepthPacking } );
```